### PR TITLE
refactor(sdk/go): use any instead empty interface

### PR DESCRIPTION
**Pull request by sashamelentyev**
Empty interface says nothing. For this case in Go1.18 have any type. It's only cosmetic change. And code with any looks cleaner than interface{}

I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

### CODE REVIEW

**Comment by verytable** [comment]
Could, you, please provide some motivation behind this change.

File: yt/go/yt/internal/params.go [hunk 1]
```diff
@@ -2027,13 +2027,13 @@ func (p *ReadTableParams) AccessTrackingOptions() **yt.AccessTrackingOptions {
 type StartOperationParams struct {
 	verb    Verb
 	opType  yt.OperationType
-	spec    interface{}
```
Comment (verytable) [inline yt/go/yt/internal/params.go:2030]: This is a generated code and the generator should be fixed as well.

One should not modify this file manually and always use generator.

**Comment by sashamelentyev** [comment]
@verytable empty interface says nothing. For this case in Go1.18 have `any` type. It's only cosmetic change. And code with any looks cleaner than interface{}

**Comment by verytable** [comment]
I'm just advocating the cleanliness of the description.

File: yt/go/yt/internal/params.go (hunk 1 above)
Comment (sashamelentyev) [reply yt/go/yt/internal/params.go]: Remove changes from this file

Comment (verytable) [reply yt/go/yt/internal/params.go]: Why not fix generator?)

Comment (verytable) [reply yt/go/yt/internal/params.go]: We'd prefer to have all client code converted at once.

**Pull request closed without merging**